src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.builds

```diff
@@ -2,13 +2,27 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <!-- todo: use default values for TargetGroup and TestTFMs once changed to current release -->
     <Project Include="System.Security.Cryptography.Algorithms.Tests.csproj">
+      <TargetGroup>netstandard1.6</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
```

Comment:
You don't need to specify `TargetGroup` because it is the default in your .csproj. Similarly you don't need to specify `TestTFMs` when it is netcoreapp1.0 as that is also the default in the repo.